azure-servicebus/HISTORY.rst

```diff
@@ -3,6 +3,16 @@
 Release History
 ===============
 
+0.50.0 (2018-01-11)
++++++++++++++++++++
+
+**Breaking changes**
```

Comment:
I think it deserves at least a "features" with "messaging is now using AMQP" ish sentence. I couldn't imagine a release note without mention of it, even if it's internal :)

Reply:
Yep sure!